### PR TITLE
Solution of copy file

### PR DESCRIPTION
**Pull request by svyshcho**

### CODE REVIEW

File: app/main.py [hunk 1]
```diff
@@ -1 +1,18 @@
-# write your code here
+
+
+def copy_file(command: str) -> None:
+    names = command.split()
+    if names[1] == names[2] or\
+            len(command.split()) != 3 or\
+            command.split()[0] != "cp":
```
Comment (olena-hul) [inline app/main.py:7]: Check command length first
Then use unpacking like `command, source_file, target_file = names`
Then check command name and files

File: app/main.py [hunk 2]
```diff
@@ -1 +1,18 @@
...
+        return
+    with open(names[1], "r") as f_r, open(names[2], "w") as f_w:
```
Comment (olena-hul) [inline app/main.py:9]: `f_r` and `f_w` are not descriptive variable names

File: app/main.py [hunk 3]
```diff
@@ -1 +1,18 @@
...
+        arr_data = f_r.readlines()
+        arr_data_checked = []
+        for i in arr_data:
+            arr_data_checked.append(i.strip("\n"))
+        for line in arr_data_checked:
+            f_w.write(line + "\n")
```
Comment (olena-hul) [inline app/main.py:15]: Why do you remove `\n` symbol and then add it again?

You can just read the content of the one file and write it to another file without any modifications

File: app/main.py [hunk 4]
```diff
@@ -1 +1,18 @@
...
+
+
+copy_file("cp test1.txt file8.txt")
```
Comment (olena-hul) [inline app/main.py:18]: Remove this line

File: app/main.py (hunk 3 above)
Comment (svyshcho) [reply app/main.py:15]: Done

File: app/main.py (hunk 2 above)
Comment (svyshcho) [reply app/main.py]: Done

File: app/main.py (hunk 1 above)
Comment (svyshcho) [reply app/main.py:7]: Done

File: app/main.py [hunk 5]
```diff
@@ -1 +1,12 @@
-# write your code here
+
+
+def copy_file(command: str) -> None:
+    names = command.split()
+    if names[1] == names[2] or\
+            len(command.split()) != 3 or\
+            command.split()[0] != "cp":
```
Comment (EdAlekseiev) [inline app/main.py:7]: You must change the order of checks:
1. Check that command `names` contains only 3 elements
2. Check that first element of `names` is "cp"
3. Check that the old name of file doesn't equal to the new name
Also, don't use command.split() a few times, please.

**Review by EdAlekseiev — CHANGES_REQUESTED** [review]

**Review by EdAlekseiev — APPROVED** [review]
Good job

**Pull request closed without merging**